content/03-reference/01-tools-and-interfaces/03-prisma-migrate.mdx

```diff
@@ -71,13 +71,13 @@ prisma migrate save --experimental
 prisma migrate up --experimental
 ```
 
-The first command _saves_ a new migration to the file system and updates the `_Migration` table. It stores information about the migration in a dedicated directory inside the `migrations` directory. Each migration that is stored has its own `README.md` file which contains detailled information about the migration (e.g. the generated SQL statements which will be executed when you run `prisma migrate up`).
+The first command _saves_ a new migration to the `prisma/migrations` directory in the file system of your project and updates the `_Migration` table in your database. Each time you run this command to save a new migration, it creates a dedicated directory inside of `prisma/migrations` for that specific migration, which will have its own `README.md` file containing detailed information about the migration (e.g. the generated SQL statements which will be executed when you run `prisma migrate up`).
```

Comment:
@nikolasburk I found this original sentence to be a little clunky/confusing just because of how many times we repeat words like migration and directory, especially if someone is brand new to Prisma.

So I took a pass at trying to make it read a little more smoothly/clearly. Let me know if you think this helps, or if perhaps this is just not necessary.